BD-2800: Clarify user email subscription (#6846)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/best_practices/managing_email_subscriptions.md

## Unsubscribed email addresses

> Braze will automatically unsubscribe any user who manually unsubscribes from your email through a [custom footer]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer). If a user has marked one or more of your emails as spam, Braze will only send transactional emails to this user and resume normal email sending if the user updates their email address.

Reviewer: Small suggestion to split this last sentence into two to avoid a potentially confusing second half!
```suggestion
Braze will automatically unsubscribe any user who manually unsubscribes from your email through a [custom footer]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer). If a user has marked one or more of your emails as spam, Braze will only send transactional emails to this user. If the user updates their email address, normal email sending will resume.
```

> Braze will automatically unsubscribe any user that either manually unsubscribes from your email through a [custom footer]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer). If a user has marked one or more of your emails as spam Braze will only send transactional emails to this user and will resume normal email sending if the user updates their email address.

Author: ```suggestion
Braze will automatically unsubscribe any user who manually unsubscribes from your email through a [custom footer]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer). If a user has marked one or more of your emails as spam, Braze will only send transactional emails to this user and resume normal email sending if the user updates their email address.
```
